repoclosure/yum.conf

```diff
@@ -183,7 +152,3 @@ failovermethod=priority
 [f29-puppet-6]
 name=f29-puppet-6
 baseurl=http://yum.puppetlabs.com/puppet6/fedora/29/x86_64/
-
-[f29-foreman-client-nightly]
```

Comment:
I've been thinking about dropping the whole of f29.

Reply:
> I've been thinking about dropping the whole of f29.

:+1:  I think we should, we are way behind on Fedora releases, and we should drop this and then be intentional about our support.